src/features/components/atoms/link/internal.tsx

```diff
@@ -19,11 +19,10 @@ import { InternalLinkType } from 'features/types'
 
 const isActiveLink = (currentPage: string, active_urls?: string[]) => {
     const pathname = typeof window !== 'undefined' ? window.location.pathname : ''
-    if (active_urls?.length) {
-        return active_urls.includes(pathname)
-    }
 
-    return pathname === currentPage
+    return active_urls?.length
+        ? active_urls.some((url) => pathname.includes(url))
+        : pathname.includes(currentPage)
```

Comment:
I know that it's not a part of bug fix, but could you please refactor `currentPage` on `current_page` everywhere?
We using snake_case for variables and camelCase for functions 🙏 
```suggestion
        : pathname.includes(current_page)
```

Reply:
yes why not sure man